client/stylesheets/_toCat.scss

```diff
@@ -2197,6 +2197,14 @@ i:hover {
     width: 70%;
 }
 
+.faqdiv h3 {
+    margin-bottom: 10px !important;
+}
+
+.faqdiv p {
+    margin-bottom: 25px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces